collector/src/main/java/com/bitmovin/analytics/BitmovinAnalytics.java

```diff
@@ -278,11 +292,22 @@ public void onAudioTrackChange() {
     EventData data = playerAdapter.createEventData();
     data.setState(playerStateMachine.getCurrentState().toString().toLowerCase());
     data.setDuration(0);
+    data.setVolume(Util.getDeviceVolume(context));
```

Comment:
Note:
Instead of adding the `setVolume` to every Event you could put this into the `createEventData` of the adapters